featuretools/primitives/standard/transform_primitive.py

```diff
@@ -320,7 +320,7 @@ class Weekday(TransformPrimitive):
     """
     name = "weekday"
     input_types = [ColumnSchema(logical_type=Datetime)]
-    return_type = ColumnSchema(logical_type=Ordinal(order=range(7)), semantic_tags={'category'})
+    return_type = ColumnSchema(logical_type=Ordinal(order=list(range(7))), semantic_tags={'category'})
```

Comment:
Is it a good rule of thumb that if a primitive's return type is `Ordinal` you need to include the `order` parameter? I notice that Ordinals in input types don't have orders

Reply:
If the return type `Ordinal` doesn't have the `order` parameter, Woodwork will error when trying to initialize on the feature matrix

I added an order to the `Year` primitive but it's very arbitrary and is just there to avoid an error